### PR TITLE
OSSMDOC-260

### PR DESCRIPTION
**Pull request by neal-timpe**
release notes for 1.1.13 and 2.0.3 z-streams

https://deploy-preview-30202--osdocs.netlify.app/openshift-enterprise/latest/support/gathering-cluster-data.html#gathering-data-specific-features_gathering-cluster-data

https://deploy-preview-30202--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#ossm-about-collecting-ossm-data_ossm-release-notes

https://deploy-preview-30202--osdocs.netlify.app/openshift-enterprise/latest/service_mesh/v2x/servicemesh-release-notes.html#new-features-red-hat-openshift-service-mesh-2-0-3

### CODE REVIEW

File: modules/ossm-about-collecting-ossm-data.adoc [hunk 1]
```diff
@@ -13,5 +13,12 @@ To collect {ProductName} data with `must-gather`, you must specify the {ProductN
 
 [source,terminal]
 ----
-$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7
+$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8
+----
+
+To collect {ProductName} data for a specific namespace with `must-gather`, you must specify the {ProductName} image and namespace. In this example, replace  `<namespace>` with your namespace.
```
Comment (tvieira) [inline modules/ossm-about-collecting-ossm-data.adoc:19]: "for a specific control plane namespace" and the other part of the paragraph "... replace `<namespace>` with your control plane namespace.

**Review by tvieira — CHANGES_REQUESTED** [review]

File: modules/ossm-about-collecting-ossm-data.adoc (hunk 1 above)
Comment (tvieira) [reply modules/ossm-about-collecting-ossm-data.adoc:19]: @neal-timpe if you have questions about what I mentioned above, please check this on maistra io docs: https://github.com/maistra/maistra.github.io/pull/176

File: modules/ossm-rn-new-features.adoc [hunk 2]
```diff
@@ -37,6 +37,10 @@ Result – If changed, describe the current user experience
 |===
 
 
+== New features {ProductName} 2.0.3
+
+This release of {ProductName} addresses Common Vulnerabilities and Exposures (CVEs) and bug fixes.
```
Comment (tvieira) [inline modules/ossm-rn-new-features.adoc:42]: In addition, this release has the following improvements:

* Added an option to must-gather to allow gathering only specific control plane namespace (OSSM-351)
* Improved performance for control planes with hundreds of namespaces

**Comment by neal-timpe** [comment]
@tvieira I made those changes you suggested. I changed the wording of the new features. Let me know if this looks right to you or if there's any more content.

**Review by tvieira — APPROVED** [review]
LGTM

File: modules/gathering-data-specific-features.adoc [hunk 3]
```diff
@@ -32,7 +32,7 @@ ifndef::openshift-origin[]
 |`registry.redhat.io/openshift-serverless-1/svls-must-gather-rhel8`
 |Data collection for OpenShift Serverless.
 
-|`registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7`
+|`registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8`
 |Data collection for Red Hat OpenShift Service Mesh.
 
 |`registry.redhat.io/rhcam-1-2/openshift-migration-must-gather-rhel8`
```
Comment (jc-berger) [inline modules/gathering-data-specific-features.adoc:38]: Sorry I can't comment on the actual line in `gathering data about specific features` section. The first `note` right before the prereqs, you say, `...add the `--image gather` argument.` Maybe say, `...add the argument, `--image gather.` 

This is totally a preference things. Whenever there's something the user might copy and paste or click, I tend to link to put it at the end of the sentence to make it easier to copy or click. Again, totally up to you and it's good as is. Simply a suggestion, thanks!

Comment (jc-berger) [inline modules/gathering-data-specific-features.adoc:38]: Again, I can't comment on the specific line. For `prereqs` the second bullet says, `The OpenShift Container Platform (oc) installed.` To be consistent with the first bullet, consider phrasing in the imperative so both bullets are actions. So the rephrase would be, `Install the OpenShift Container Platform (oc).`

File: modules/ossm-about-collecting-ossm-data.adoc [hunk 4]
```diff
@@ -13,5 +13,12 @@ To collect {ProductName} data with `must-gather`, you must specify the {ProductN
 
 [source,terminal]
 ----
-$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel7
+$ oc adm must-gather --image=registry.redhat.io/openshift-service-mesh/istio-must-gather-rhel8
+----
+
+To collect {ProductName} data for a specific control plane namespace with `must-gather`, you must specify the {ProductName} image and namespace. In this example, replace  `<namespace>` with your control plane namespace, such as `istio-system`.
```
Comment (jc-berger) [inline modules/ossm-about-collecting-ossm-data.adoc:19]: In the previous code snippet, you introduced it with a colon, `...Service Mesh image:` 

So wouldn't you need to introduce this code snippet with a colon too? So, it'd be, `...such as `istio-system`:`

File: modules/ossm-rn-fixed-issues.adoc [hunk 5]
```diff
@@ -19,6 +19,8 @@ The following issues been resolved in the current release:
 [id="ossm-rn-fixed-issues-ossm_{context}"]
 == {ProductShortName} fixed issues
 
+* link:https://issues.redhat.com/browse/OSSM-419[OSSM-419] Namespaces with similar names will all show in Kiali namespace list, even though namespaces may not be defined in Service Mesh Member Role.
```
Comment (JStickler) [inline modules/ossm-rn-fixed-issues.adoc:22]: It looks like the convention for including issue links as part of a New Feature is to put it at the end of the description and format it as "See [issue link] for more information.

https://docs.openshift.com/container-platform/4.7/release_notes/ocp-4-7-release-notes.html#ocp-4-7-rhcos

Comment (rolfedh) [inline modules/ossm-rn-fixed-issues.adoc:22]: Hi, @neal-timpe. This looks like a description of the bug from OSSM-419. I'm not familiar with how ossm handles release notes. However, a typical release notes describes the bug behavior in past tense, followed by a description of the fix in the present tense. For example, see https://docs.google.com/presentation/d/1TaI1jbV-dhYueUfElPjS_HEqLag_HexFWp85Kdst4y8/edit#slide=id.gc78664d2cb_1_47

Comment (rolfedh) [reply modules/ossm-rn-fixed-issues.adoc:22]: A naive reading of "Namespaces with similar names will all show in Kiali namespace list" under the `== {ProductShortName} fixed issues` heading suggests that you're describing the new fixed behavior.

**Pull request merged**